federatedscope/core/data/base_data.py

```diff
@@ -0,0 +1,173 @@
+import logging
+from federatedscope.core.data.utils import merge_data
+from federatedscope.core.auxiliaries.dataloader_builder import get_dataloader
+
+logger = logging.getLogger(__name__)
+
+
+class StandaloneDataDict(dict):
+    """
+        `StandaloneDataDict` maintain several `ClientData`.
+    """
+    client_cfgs = None
```

Comment:
why is `client_cfgs` a class attribute but `cfg` an object attribute? Any difference in the roles they play?